frontend/package.json

```diff
@@ -1,35 +1,22 @@
 {
-  "name": "frontend",
-  "version": "0.0.1",
+  "name": "@etalab/ouverture.data.gouv.fr",
+  "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "authors": {
-    "name": "Etalab",
-    "email": ""
-  },
-  "repository": "/frontend",
   "scripts": {
-    "dev": "vuepress dev site",
-    "build": "vuepress build site"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "docs:dev": "vitepress dev site",
```

Comment:
Is there a reason for `docs:dev`? I thought it was related to some automated doc at first :p

Reply:
Not really, this is the default scripts for new vitepress project.